js/components/mixing-view.js

```diff
@@ -140,7 +141,7 @@ export default class MixingView extends PureComponent {
             showCelsius={showCelsius}
             aPegged={aTemp}
             bPegged={bTemp}
-            forceCover={true}
+            forceCover={frozen}
```

Comment:
I think it's fine to leave this as it was - the single view Thermoscope should never be changeable by hand as far as I can tell.

Reply:
Fixed, thanks!